plugins/woocommerce-admin/client/customize-store/data/homepageTemplates.ts

```diff
@@ -1,11 +1,20 @@
+/**
+ * Internal dependencies
+ */
+import { isWooExpress } from '~/utils/is-woo-express';
+
+const introPatternWooExpress = 'woocommerce-blocks/just-arrived-full-hero';
+
 export const HOMEPAGE_TEMPLATES = {
 	template1: {
 		blocks: [
 			// Header
 			'woocommerce-blocks/header-centered-menu',
 
 			// Body
-			'woocommerce-blocks/hero-product-split',
+			isWooExpress()
+				? introPatternWooExpress
+				: 'woocommerce-blocks/just-arrived-full-hero',
```

Comment:
@albarin do you mind confirming what's the expected conditional here, please? ATM the `'woocommerce-blocks/just-arrived-full-hero'` is being used both on Woo Core and on WooExpress (`const introPatternWooExpress = 'woocommerce-blocks/just-arrived-full-hero';`)

PS: if the decision is to have `'woocommerce-blocks/just-arrived-full-hero'` on WooExpress, we also have to update the following constant to ensure AI content is generated for it: https://github.com/woocommerce/woocommerce/blob/9d86e1f038a91e190ef9f53d1042ec2d04ff633d/plugins/woocommerce/src/Blocks/AIContent/UpdatePatterns.php#L16-L26

Reply:
My bad, you are totally right, the idea is to keep the `woocommerce-blocks/hero-product-3-split` as intro in WooExpress. Just pushed a fix 🙏